### PR TITLE
Change UX of Test Version select to show current git commit and a drop down when there are new versions

### PR DESCRIPTION
**Pull request by evmiguel**
This PR is dependent on #202 

- When there are no new versions:
Screen Shot 2020-11-04 at 3 15 01 PM

- When there are new versions:
Screen Shot 2020-11-04 at 3 21 57 PM

### CODE REVIEW

File: client/components/ConfigureActiveRuns/index.jsx [hunk 1]
```diff
@@ -358,21 +359,51 @@ class ConfigureActiveRuns extends Component {
...
                 <Form className="init-box">
                     <Row>
                         <Col>
-                            <Form.Group controlId="testVersion">
-                                <Form.Label data-test="configure-run-commit-label">
-                                    Git Commit of Tests
-                                </Form.Label>
-                                {this.renderTestVersionSelect()}
-                            </Form.Group>
+                            {
```
Comment (evmiguel) [inline client/components/ConfigureActiveRuns/index.jsx:371]: This is the meat of the PR. The other changes came with lint fixes

File: client/components/ConfigureActiveRuns/index.jsx [hunk 2]
```diff
@@ -539,21 +541,51 @@ class ConfigureActiveRuns extends Component {
...
+                                null
+                            }
                         </Col>
                     </Row>
+                    {renderedTestVersions !== null ? (
+                        <Row>
+                            <Col>
+                                <Form.Group controlId="testVersion">
+                                    <Form.Label data-test="configure-run-commit-label">
+                                        Git Commit of Tests
```
Comment (spectranaut) [inline client/components/ConfigureActiveRuns/index.jsx:582]: This should be "select a different commit"

**Review by spectranaut — APPROVED** [review]
Looks good other than that one string!

**Review by spectranaut — APPROVED** [review]
Perfect

**Pull request merged**